docs/getting-started/quickstart-tt-debugging.md

```diff
@@ -0,0 +1,187 @@
+---
+sidebar_position: 3
+title: Time Travel Debugging Quickstart
+---
+
+Now that we've learned a little about programming DBOS, let's learn how to use the DBOS Time Travel Sebugger!
+
+This tutorial assumes you have [Visual Studio Code](https://code.visualstudio.com/) installed.
+Please see the VS Code documentation for [installation instructions](https://code.visualstudio.com/docs/setup/setup-overview) if necessary. 
+
+:::info
+If you're not a VS Code user, there is an advanced tutorial for [Time Travel Debugging with DBOS CLI](../cloud-tutorials/timetravel-debugging#time-travel-with-dbos-sdk-cli-non-vs-code-users).
+:::
+
+Additionally, this tutorial builds on the [DBOS Quickstart](./quickstart).
+For convenience, we recommend [creating a DBOS Cloud account](./quickstart#deploying-to-dbos-cloud) and 
+[provisioning a DBOS Cloud database instance](./quickstart#provisioning-a-cloud-database-instance) before starting this tutorial.
+
+### Installing the DBOS Time Travel Debugging VS Code Extension
+
+Before we use the DBOS Time Travel debugger, we need to install its VS Code extension.
+
+To install the extension, navigate to the [extension's web page](https://marketplace.visualstudio.com/items?itemName=dbos-inc.dbos-ttdbg) and press the "Install" button. 
+This will launch VS Code and open the DBOS Time Travel Debugger extension page inside the IDE. 
+From there, select the "Install" button to install the extension.
+
+![Installing the DBOS Extension](./assets//ttdbg-install.png)
+
+:::info
+You can also install the extension by opening the [Extension Marketplace](https://code.visualstudio.com/docs/editor/extension-marketplace) 
+inside VS Code (default keybinding: Ctrl+Shift+X / ⇧⌘X) and searching for "DBOS".
+:::
+
+### Deploy the Programming Quickstart App to DBOS Cloud
+
+This tutorial uses with the application you built for the [Programming Quickstart](./quickstart-programming). 
+If you didn't finish that tutorial, the code is available on [GitHub](https://github.com/dbos-inc/dbos-demo-apps/tree/main/greeting-emails).
```

Comment:
I think we can put a `dbos create` command here:
```suggestion
If you didn't finish that tutorial, you can create the app by running the `[dbos create]` command
```

Reply:
Is it enough to tell the reader to `dbos create` w/o also repeating instructions for adding the code from programming quickstart ?